docs/bonding-intro.md

```diff
@@ -0,0 +1,255 @@
+What are groups
+===============
+
+A Group is an entity that binds multiple sockets and it is required to
+establish a "bonded connection". Groups can be then used the same way as
+sockets for performing a transmission. It is then in general stated that a
+group is connected as long as at least one member-socket connection is alive,
+and as long as this state lasts, some member connections may get broken and
+new member connections can be established.
+
+Groups are fully flexible. There's no limitation how many single connections
+they can use as well as when you want to establish a new connection. On the
+other hand, broken connections are not automatically reestablished. The
+application should track the existing connections and reestablish broken ones
+if needed. But then, the application is also free to keep as many links as it
+wants, including adding new links to the group while it is being used for
+transmission, or removing links from the list if they are not to be further
+used.
+
+How the links are exactly utilized within the group, it depends on the group
+type. The simplest type, broadcast, utilizes all links at a time to send the
+same data.
```

Comment:
How links are utilized within a group depends on the group type. 
The simplest type, broadcast, utilizes all links at once to send the
same data.